content/en/guides/features/loading.md

```diff
@@ -68,7 +68,7 @@ questions:
 
 Out of the box, Nuxt.js gives you its own loading progress bar component that's shown between routes. You can customize it, disable it or even create your own loading component.
 
-## Customising the Progress Bar
+## Customizing the Progress Bar
```

Comment:
This change would break old links pointing to https://nuxtjs.org/guides/features/loading#customising-the-progress-bar. Is that okay?

Reply:
Could you check is we link to it in the docs and update it please?

Reply:
I think he means any links that are linking from google results but I don't think its an issue with the anchor tag and if it is we can add a redirect in netlify to cover it but personally I don't think we need to and just need to update any links across the site.

Reply:
Exactly, only the hash would be different. Probably not a big deal, but I wanted to mention. 

There are no other occurrences of "customising" in this repo, so no links need to get updated.